internal/controllers/common/resource_definitions/resource_definitions.go

```diff
@@ -464,6 +484,24 @@ func NewPodForReports(cr *operatorv1beta1.Cryostat, imageTags *ImageTags, tls *T
 			Path:   "/health",
 		},
 	}
+
+	var nodeSelector map[string]string
+	var affinity *corev1.Affinity
+	var tolerations []corev1.Toleration
+
+	if cr.Spec.ReportOptions.SchedulingOptions != nil {
```

Comment:
`cr.Spec.ReportOptions.SchedulingOptions` is pretty long. Refactoring this to a local variable would make this block more readable.